### DISCUSSION TITLE
Will case count data still be updated?

### DISCUSSION THREAD

**Discussion by joverlee521** [owid/covid-19-data#2782] [General]
John Hopkins [announced](https://github.com/CSSEGISandData/COVID-19/issues/6577) they will stop updating COVID-19 data on March 10, 2023. 
Will OWID find a new source for case counts or will you also stop updating case data?

**Comment by edomt** [comment]
Hi @joverlee521 

We are working on evaluating alternatives to the JHU dataset. We'll be communicating on this over the next couple of weeks.
